gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -0,0 +1,140 @@
+import { Result, WriteResponse } from '@chainlink/gauntlet-core'
+import { TransactionResponse, SolanaCommand, RawTransaction } from '@chainlink/gauntlet-solana'
+
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+import { PublicKey, SYSVAR_RENT_PUBKEY, Keypair } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
+
+// enum ACTIONS {
+//   create = 'create',
+//   approve = 'approve',
+//   execute = 'execute',
+// }
+
+export const wrapCommand = (command) => {
+  return class Multisig extends SolanaCommand {
+    command: SolanaCommand
+    multisigAddress
+    multisig
+    address
+    program
+    rawTx
+
+    static id = `${command.id}`
+
+    constructor(flags, args) {
+      super(flags, args)
+      logger.info(`Serum multisig wrapping of ${command.id}`)
+
+      this.command = new command(flags, args)
+      this.command.invokeMiddlewares(this.command, this.command.middlewares)
+    }
+
+    createProposal = async (rawTx: RawTransaction) => {
```

Comment:
Please type the expected returned type (`Promise<{ tx: string, transaction: PublicKey }>`)